libs/core/src/components/sage-switch/test/sage-switch.e2e.ts

```diff
@@ -0,0 +1,83 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('sage-switch', () => {
+  it('renders a checked input when toggled', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<sage-switch></sage-switch>');
+    const component = await page.find('sage-switch');
+    expect(component).toHaveClass('hydrated');
+
+    const el = await page.find('sage-switch >>> input');
+    let value = await el.getProperty('checked');
+    expect(value).toBe(false);
+
+    component.setProperty('checked', 'true');
+    await page.waitForChanges();
+    value = await el.getProperty('checked');
```

Comment:
My personal opinion and I'll let others chime in. I prefer a newline (empty line break) after `await page.waitForChanges`. For me, it indicates that we are working with a new instance of the page.

Reply:
I like this. If specs are instructions, that makes line breaks new paragraphs. Maybe take it one step further for consistency by separating any leading `await` operator use.